app/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
     testImplementation 'androidx.arch.core:core-testing:2.0.0'
-    implementation 'com.squareup.retrofit2:retrofit:2.7.1'
-    implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
```

Comment:
Made sure dependencies like retrofit are up to date
Source: https://nullbeans.com/9-reasons-for-keeping-software-dependencies-up-to-date/